packages/flutter_test/lib/src/binding.dart

```diff
@@ -611,6 +642,24 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   late StackTraceDemangler _oldStackTraceDemangler;
   FlutterErrorDetails? _pendingExceptionDetails;
 
+  _MockMessageHandler? _announcementHandler;
```

Comment:
Why do we need this variable? Could you just use _handleAnnouncementMessage directly wherever it is used?